File: corpus/py/F;0282.py
int Valid = 0
while Valid == 0:
    print("Digite o Código do Produto: ") 
    int prod = int(input())
    if prod == 0:
        Valid = 1
    else:
        print("Digite a quantidade do produto: ")
        int quant = int(input())
        print()

File: corpus/2017q1/_p2q2_ok/_check/F;0282.java

import java.util.Scanner;
public class P2Ex02 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int Valid = 0;

        while (Valid == 0){
            System.out.print("Digite o Código do Produto: ") ;
            int prod = sc.nextInt();
                if(prod == 0){
                    Valid = 1;
                }
                else{
                    System.out.print("Digite a quantidade do produto: ");
                    int quant = sc.nextInt();
                    System.out.println();
                }
           }
        }
    }
